src/components/Layout/Header.tsx

```diff
@@ -72,14 +72,16 @@ const Header: React.FC<HeaderProps> = ({ hideSearchBar = false }) => {
       // mobileNav={
       //   <TabMenu
       //     tabs={tabs}
-      //     contents={[<LeftSidebar inHeader key="nav-mobile-documentation-tab" />, null]}
+      //     contents={[<LeftSidebar inHeader key="nav-mobile-documentation-tab" />, <ExamplesList key="nav-mobile-examples-tab" />]}
```

Comment:
Just wondering why this is still commented out. If we merge and push this to production, will it remain commented? Would it be easier to add a feature flag instead?

Reply:
@aralovelace we need to land #2383 first _and_ update the website to routes `/examples` to docs

Reply:
We also need to do the sandpack stuff as examples wont be considered complete until then

Reply:
okay noted @jamiehenson  @kennethkalmer  I cant approved it but all good for me